cal/cal/static/js/app.js

```diff
@@ -180,20 +179,83 @@ function TagsDetailCtrl($scope, $http) {
         xAxis: {
           axisLabel: 'Date',
           tickFormat: function(d) {
-            return d3.time.format('%m/%d/%y')(d)
-          }
+                          return d3.time.format('%m/%d/%y')(d)
+                      }
         },
         yAxis: {
           axisLabel: 'Hours',
-          tickFormat: function(d) {
+          tickFormat: function(d){
             return d3.format('.02f')(d);
           },
           axisLabelDistance: -10,
         },
-        forceY: [0, max_hour + 1],
+        forceY: [0, maxYValue+1],
```

Comment:
Add spaces between +. Just saying what Max would say, he got mad at me for not adding spaces around division.